### PR TITLE
Dpm #ticket trivia

### PR DESCRIPTION
**Pull request by DanielPadillaM**
trivia changes

### CODE REVIEW

File: src/js/components/triviaQuestions.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
+import {createQuestion} from '../view/createQuestion';
+import {Pregunta} from '../models/Pregunta';
+
+export const triviaContainer = document.querySelector('.pokemonTrivia');
+triviaContainer.addEventListener('click',(event)=>{
+
+    let questionTitle = document.querySelector('.pregunta1__title')
```
Comment (ldpmcasebook) [inline src/js/components/triviaQuestions.js:8]: try to not hardcode string they can change and they are easy to lose
try 
```
 questions: {
   one: 'Muy bien respuesta correcta!, fue muy facil sigamos con la siguiente",'Cual de est',
    two: '',
  ....
}
```
they will not change also this is a strategy used when you will do multilanguage if later one we want texts in spanish and english they will be stored in a same place

File: src/js/models/Pregunta.js [hunk 2]
```diff
@@ -0,0 +1,13 @@
+import {createQuestion} from "../view/createQuestion"
+export class Pregunta {
+    constructor(respuesta,questions){
+        let result = document.querySelector('.respuesta')
+        result.textContent= respuesta
+        result.addEventListener('click',()=>{
+            let question = document.querySelector('.pregunta1')
```
Comment (ldpmcasebook) [inline src/js/models/Pregunta.js:7]: when a variable isn't reassigned you must use const, some things like linters make you aware of that and give you suggestions you could try to integrate eslint in your project we will configure it this weekend.

File: src/js/models/Trivia.js [hunk 3]
```diff
@@ -0,0 +1,24 @@
+import {createQuestion} from "../view/createQuestion"
+export const triviaStart = document.querySelector('.pokemonTrivia__container');
+export class Trivia {
+    constructor(parent){
+        const pokemonTrivia__title = document.createElement('h1')
+        pokemonTrivia__title.classList.add("pokemonTrivia__title")
```
Comment (ldpmcasebook) [inline src/js/models/Trivia.js]: in javascript we always use conventions in this case is camelCase
words will be like pokemonTriviaTitle never with dashes or undercores or anything else it is a convention that we must follow always in code

File: src/js/models/Trivia.js [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+        pokemonTrivia__title.textContent= 'Bienvenido a la trivia Pokemon'
+        const pokemonTrivia__text = document.createElement('p');
+        pokemonTrivia__text.classList.add('pokemonTrivia__text')
+        pokemonTrivia__text.textContent = "Puedes responder correctamente las siguientes preguntas?"
+        const triviaButton = document.createElement('button');
+        triviaButton.classList.add('pokemonTrivia__button');
+        triviaButton.textContent= 'Iniciar Trivia Pokemon'
+        triviaButton.addEventListener('click',()=>{
+            triviaStart.remove();
+            createQuestion('Cual de estos Pokemones es tipo Planta');
```
Comment (ldpmcasebook) [inline src/js/models/Trivia.js]: same as text constant suggestion on top, store it under constants file an import it here.

File: src/js/models/pokemonBattle/pokemonField.js [hunk 5]
```diff
@@ -0,0 +1,6 @@
+export const createPokemonfield = ()=>{
+    let pokemonfield = document.createElement('div')
+    pokemonfield.classList.add('pokemonField')
+const body = document.querySelector('.battle')
```
Comment (ldpmcasebook) [inline src/js/models/pokemonBattle/pokemonField.js]: identation const body should start the same as pokemonfield, also rename pokemonfield to pokemonField conventions always conventions

File: src/js/models/pokemonBattle/pokemonField.js [hunk 6]
```diff
@@ -0,0 +1,6 @@
+export const createPokemonfield = ()=>{
+    let pokemonfield = document.createElement('div')
```
Comment (ldpmcasebook) [inline src/js/models/pokemonBattle/pokemonField.js]: pokemonField isn't reassigned use const

File: src/js/models/randomPType.js [hunk 7]
```diff
@@ -0,0 +1,21 @@
...
+export const randonNumber= (min, max)=> Math.round(Math.random() * (max - min) + min);
+
+export const randomPokemonType = (pokemonTypeUrl)=>{
+    return axios.get(pokemonTypeUrl)
+    .then((pokemon)=>{
+        const pokeIndex = pokemon.data.pokemon.length-1;
+        const randomPokemon = randonNumber(0,pokeIndex)
+        const randomUrl = pokemon.data.pokemon[randomPokemon].pokemon.url
+        return axios.get(randomUrl)
+        .then((pokemon)=>{
```
Comment (ldpmcasebook) [inline src/js/models/randomPType.js:12]: great work!

File: src/js/view/pokemonImageContainer.js [hunk 8]
```diff
@@ -0,0 +1,9 @@
+export const pokemonImageContainer= (className,img,imageClass)=>{
+    let div = document.createElement('div')
+    div.classList.add(className)
+    let image = document.createElement('img')
```
Comment (ldpmcasebook) [inline src/js/view/pokemonImageContainer.js]: same const suggestions

File: src/sass/layout/_nav.scss [hunk 9]
```diff
@@ -1,9 +1,9 @@
 nav{
-    grid-column: 2/3;
-    grid-row: 1/2;
+    display: flex;
+    justify-content: center;
+    align-items: center;
 }
 .barra{
-    display: flex;
     list-style: none;
```
Comment (ldpmcasebook) [inline src/sass/layout/_nav.scss:7]: remenber to always use english convention from now on, there's no places where they write code in spanish

File: src/sass/layout/_pokemonField.scss [hunk 10]
```diff
@@ -0,0 +1,7 @@
+.pokemonField{
```
Comment (ldpmcasebook) [inline src/sass/layout/_pokemonField.scss:1]: in sass or css isn't neccesary to use camelCase I personally do, but here if you will always use BEM convention or any convention be consistent not use it in some places and in other doesn't.

File: src/js/components/triviaQuestions.js (hunk 1 above)
Comment (DanielPadillaM) [reply src/js/components/triviaQuestions.js:8]: done

File: src/js/models/Pregunta.js (hunk 2 above)
Comment (DanielPadillaM) [reply src/js/models/Pregunta.js:7]: done

File: src/js/models/Trivia.js (hunk 3 above)
Comment (DanielPadillaM) [reply src/js/models/Trivia.js]: done

File: src/js/models/Trivia.js (hunk 4 above)
Comment (DanielPadillaM) [reply src/js/models/Trivia.js]: done

File: src/js/models/pokemonBattle/pokemonField.js (hunk 5 above)
Comment (DanielPadillaM) [reply src/js/models/pokemonBattle/pokemonField.js]: done

File: src/js/models/pokemonBattle/pokemonField.js (hunk 6 above)
Comment (DanielPadillaM) [reply src/js/models/pokemonBattle/pokemonField.js]: done

File: src/js/view/pokemonImageContainer.js (hunk 8 above)
Comment (DanielPadillaM) [reply src/js/view/pokemonImageContainer.js]: done

File: src/sass/layout/_nav.scss (hunk 9 above)
Comment (DanielPadillaM) [reply src/sass/layout/_nav.scss:7]: done

**Review by LuisPadillaM — APPROVED** [review]

**Pull request merged**